packages/shopping/src/controllers/user.controller.ts

```diff
@@ -58,18 +74,36 @@ export class UserController {
       },
     },
   })
-  async create(@requestBody() user: User): Promise<User> {
+  async create(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(NewUserRequest, {
+            title: 'NewUser',
```

Comment:
Should we exclude 'id' in the schema to be consistent with the type? The type for data is `newUserRequest: Omit<NewUserRequest, 'id'>,`

Reply:
Good catch.

We must allow `id` in the payload, see the test below and also the docs. I'll update the type to `newUserRequest: NewUserRequest`.